### DISCUSSION TITLE
Updating via tesseract

### DISCUSSION THREAD

**Discussion by rin67630** [dynobo/normcap#166] [Q&A] [answered]
Tank you for that life saver!
As a guy, who writes a lot of documentation, I loooove it !

This app is dependent on Tesseract. 
Does is update automatically if I update/reinstall Tesseract?
Sorry for my noob question.

**Comment by dynobo** [comment, marked as answer]
Hi @rin67630, 
Updating Tesseract will never update NormCap itself, but NormCap might use the new Tesseract version, depending on which installation type of NormCap you've chosen:

#### Python package (pip install normcap)

The Python package of NormCap does not include Tesseract, instead it uses the Tesseract version installed on the system. (That's why installing Tesseract is a pre-requisite of installing NormCap as Python package). 

- To update Tesseract, you'll have to update your  systems Tesseract installation. NormCap will then automatically use this updated version (it always uses the version that is called if you run `tesseract` in the command line).
 - To update NormCap itself, you'll have to do a `pip install --upgrade normcap` manually. It will not be updated, if you update Tesseract.

#### Prebuild packages (exe-installer, *.AppImage):

In the prebuild packages, Tesseract is shipped together with NormCap. NormCap will always use this shipped Tesseract version, which probably is (a bit) older than the most current one. It is not affected by any other Tesseract installation/updates.

- To update NormCap, you'll have to download and install a new prebuild package.
- Whether or not this also updates the included Tesseract version depends on the release (see [CHANGELOG.md](https://github.com/dynobo/normcap/blob/main/CHANGELOG.md)). So far, only Tesseract 4 has been shipped, but I'm currently working on a bigger update that might switch to Tesseract 5

Does this help you? :-)
